Form/Type/VichImageType.php

```diff
@@ -19,6 +19,12 @@
  */
 class VichImageType extends VichFileType
```

Comment:
Why on VichImageType? Don't you think that such change would fit on VichFileType too?

Reply:
This code is meant only to generate the value of `image_uri` (which is specific to `VichImageType`, not `VichFileType`). So it feels in the right place to me.

Reply:
I guess that the problem described in related issue can apply also to a generic file: the user want to display a link to file, and link href attribute is technically the same as image src attribute (a path, in the end)

Reply:
You're right, the option applies only to VichImageType